### PR TITLE
feat(servicecatalog): graduate to stable 🚀

### PR DESCRIPTION
**Pull request by arcrank**
We will move library to stable, and promise semantic versioning for APIs.

We have no expected breaking changes at this time, and no reported bugs.

**Feel free to report or question any current APIs**

----

### All Submissions:

* [X] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md)

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by kaizen3031593** [comment]
@rix0rrr we should probably discuss this before graduating this module. We should have a more stringent process for graduating construct libraries. @arcrank stay tuned :).

**Comment by arcrank** [comment]
I noticed for some older PRs that there was a checklist included that had items like no p0 bugs in past 4 weeks, cleared lint exceptions etc.. If you have a current one for that we can go over it since I'm not sure if all of that will apply. 

We've already reviewed our APIs internally and have no plans for any behavior change in our Cfn Handlers either.

**Comment by arcrank** [comment]
Want to circle back on this.  We had review. There were two takeaways I had

1. Minor doc nits/updates.  These were addressed and merged in
2. Ideas around adding default policies on sharing a portfolio.  I went back to our team and PMs to understand usage a little better and I think that for now, we don't see value in adding some limited subset of resource policies to shared users.  Read only permissions are not very helpful or useful, and due to how we make our own ids for provisioning users require some wildcard permissions to actually provision a product.

**Comment by rix0rrr** [comment]
The `constrainTagUpdates` API is still confusing to me (call `constrain` -> actually means "allow"). Maybe the same is true for other APIs. Can you find a synonym for that? `configure` maybe?

**Review by kaizencc — APPROVED** [review]
🎉

**Review by kaizencc — DISMISSED** [review]

File: packages/@aws-cdk/aws-servicecatalog/README.md [hunk 1]
```diff
@@ -351,7 +342,7 @@ as well as in any end user accounts that wish to provision a product with the la
 You can do this by passing in the role with an explicitly set name:
 
 ```ts
-import * as iam from '@aws-cdk/aws-iam';
+import * as iam from '@aws-cdk/aws-iam'
```
Comment (arcrank) [inline packages/@aws-cdk/aws-servicecatalog/README.md:345]:
```suggestion
import * as iam from '@aws-cdk/aws-iam';
```

**Review by kaizencc — APPROVED** [review]

**Review by kaizencc — DISMISSED** [review]

**Review by kaizencc — APPROVED** [review]

**Pull request merged**